tdrs-backend/tdpservice/data_files/admin/admin.py

```diff
@@ -135,6 +145,28 @@ def queryset(self, request, queryset):
             else:
                 return queryset
 
+    class FRA_AccessFilter(admin.SimpleListFilter):
```

Comment:
This filter will appear if user has FRA access and is Admin

Reply:
view for user without feature flag
![Screenshot 2025-02-07 144711](https://github.com/user-attachments/assets/90cbe0e2-ae27-4f6e-bcdc-75fde69e129a)

view for user with feature flag

![Screenshot 2025-02-07 144815](https://github.com/user-attachments/assets/39034854-7a6a-4c03-8582-b7eb3d35ec09)